src/pages/search.tsx

```diff
@@ -55,74 +57,68 @@ const Search = ({
         </div>
       </PageHeader>
       <GridContainer>
-        <>
-          <div className={styles.pageTitle}>
-            <h2>
-              {resultString} for ‘{query}’
-            </h2>
-          </div>
+        <div className={styles.pageTitle}>
+          <h2>
+            {resultString} for ‘{query}’
+          </h2>
+        </div>
+
+        <Grid row gap="lg">
+          <Grid col="auto">
+            {/* <EPubsCard query={query} /> */}
```

Comment:
I left this in because I'm not sure what to do with it? I didn't see it anywhere in the design, but I imagine we want to keep it around. I'll bring it up at the next team sync.